2021-11-18 23:22:40,900 INFO    MainThread:12446 [wandb_setup.py:_flush():71] setting env: {}
2021-11-18 23:22:40,901 INFO    MainThread:12446 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-11-18 23:22:40,901 INFO    MainThread:12446 [wandb_init.py:_log_setup():357] Logging user logs to /home/nicolas/workspace_josue/Pytorch-UNet/wandb/run-20211118_232240-3puvw5pw/logs/debug.log
2021-11-18 23:22:40,901 INFO    MainThread:12446 [wandb_init.py:_log_setup():358] Logging internal logs to /home/nicolas/workspace_josue/Pytorch-UNet/wandb/run-20211118_232240-3puvw5pw/logs/debug-internal.log
2021-11-18 23:22:40,902 INFO    MainThread:12446 [wandb_init.py:init():390] calling init triggers
2021-11-18 23:22:40,902 INFO    MainThread:12446 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-18 23:22:40,903 INFO    MainThread:12446 [wandb_init.py:init():435] starting backend
2021-11-18 23:22:40,903 INFO    MainThread:12446 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-18 23:22:40,949 INFO    MainThread:12446 [backend.py:ensure_launched():199] starting backend process...
2021-11-18 23:22:40,970 INFO    MainThread:12446 [backend.py:ensure_launched():205] started backend process with pid: 12466
2021-11-18 23:22:40,970 INFO    MainThread:12446 [wandb_init.py:init():444] backend started and connected
2021-11-18 23:22:40,972 INFO    MainThread:12446 [wandb_init.py:init():503] updated telemetry
2021-11-18 23:22:41,027 INFO    MainThread:12446 [wandb_init.py:init():533] communicating current version
2021-11-18 23:22:41,310 INFO    MainThread:12446 [wandb_init.py:init():538] got version response 
2021-11-18 23:22:41,310 INFO    MainThread:12446 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-18 23:22:41,636 INFO    MainThread:12446 [wandb_init.py:init():576] starting run threads in backend
2021-11-18 23:22:42,576 INFO    MainThread:12446 [wandb_run.py:_console_start():1693] atexit reg
2021-11-18 23:22:42,576 INFO    MainThread:12446 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-11-18 23:22:42,577 INFO    MainThread:12446 [wandb_run.py:_redirect():1572] Redirecting console.
2021-11-18 23:22:42,578 INFO    MainThread:12446 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-18 23:22:42,578 INFO    MainThread:12446 [wandb_init.py:init():603] run started, returning control to user process
2021-11-18 23:22:42,578 INFO    MainThread:12446 [wandb_run.py:_config_callback():911] config_cb None None {'epochs': 5, 'batch_size': 5, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_size': [250.0, 250.0], 'amp': True}
2021-11-18 23:22:42,932 INFO    MainThread:12446 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2021-11-18 23:22:42,932 INFO    MainThread:12446 [wandb_run.py:_restore():1635] restore
2021-11-18 23:22:45,624 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 862
}

2021-11-18 23:22:45,868 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 862
}

2021-11-18 23:22:46,659 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 2973
}

2021-11-18 23:22:46,763 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 14131
}

2021-11-18 23:22:46,865 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14131
  total_bytes: 14131
}

2021-11-18 23:22:46,968 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14131
  total_bytes: 14131
}

2021-11-18 23:22:47,071 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14131
  total_bytes: 14131
}

2021-11-18 23:22:47,173 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14131
  total_bytes: 14131
}

2021-11-18 23:22:47,276 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14131
  total_bytes: 14131
}

2021-11-18 23:22:47,379 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14131
  total_bytes: 14131
}

2021-11-18 23:22:47,524 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14131
  total_bytes: 14131
}

2021-11-18 23:22:48,122 INFO    MainThread:12446 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14131
  total_bytes: 14131
}
local_info {
}

2021-11-18 23:22:49,181 INFO    MainThread:12446 [wandb_run.py:_append_files():2061] logging synced files
